### DISCUSSION TITLE
[Feature] Share intent on mobile to allow manual upload to Immich

### DISCUSSION THREAD

**Discussion by eltiel** [immich-app/immich#4399] [Feature Request]
### The feature

To be able to do one-off manual upload to Immich by "sharing" a photo and selecting Immich app as the handler.

This would be useful in cases where the photo is not located in the synced folder.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2659
